modelstore/storage/gcloud.py

```diff
@@ -116,14 +142,22 @@ def _pull(self, source: str, destination: str) -> str:
         logger.info("Downloading from: %s...", source)
         file_name = os.path.split(source)[1]
         destination = os.path.join(destination, file_name)
-        bucket = self.client.get_bucket(self.bucket_name)
+        if self.client.project is None:
```

Comment:
Small nit (non blocker); we're using 

```
if self.client.project is None
```

To mean "if we're accessing a public bucket anonymously;" which is not very explicit when we're doing a `is None` check. I wonder whether it would be better to extract this into a `self.is_read_only()` or something like that?

Reply:
Good point, I'll make this more clear.

I tried to find out whether there is a way to tell if a client is an anonymous client but there seems to be no flag/property one could check except the project being `None`.